api/godog/json/matcher_test.go

```diff
@@ -26,6 +26,20 @@ func TestMatcherNestedMulti(t *testing.T) {
 	require.Nil(t, Match(json, ".keyA.keyC", "valueC"))
 }
 
+func TestMatcherNumbers(t *testing.T) {
+	json := []byte(`{"key" : 12345}`)
+	require.Nil(t, Match(json, ".key", `12345`))
+
+	json = []byte(`{"key" : 123.456}`)
+	require.Nil(t, Match(json, ".key", `123\.456`))
```

Comment:
is it worth to add a regexp so that you can match on `123.456` instead?

Reply:
It is...